.gitignore

```diff
@@ -299,3 +299,8 @@ sketch
 static
 
 # End of https://www.toptal.com/developers/gitignore/api/flask,react,macos,node
+
+# Elastic Beanstalk Files
+.elasticbeanstalk/*
+!.elasticbeanstalk/*.cfg.yml
+!.elasticbeanstalk/*.global.yml
```

Comment:
we want these config files to be ignored? I thought it would be helpful to version these.

Reply:
This is `.gitignore` speak for "Ignore everything in the `.elasticbeanstalk` folder, *except* files that end with `.cfg.yml` or `.global.yml`." So the config files will be brought under version control. These lines were added by the `eb` command itself :)

I know EB uses the folder for temp files during deploy, for example.

Reply:
oh haha you're right! It's been a long day and i misread the `.gitignore `